### PR TITLE
Project submitted

### PR DESCRIPTION
**Pull request by tejbans**

### CODE REVIEW

File: bin/tictactoe [hunk 1]
```diff
@@ -1,3 +1,8 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+TicTacToeCLI.new.call
```
Comment (randallreedjr) [inline bin/tictactoe]: Beautiful!

File: lib/players/computer.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      self.token == "X" ? "O" : "X"
+   end
+ 
+   def move(board)
+     return "5" if !board.taken?(5)
+     best_option(board)
+   end
+ 
+   def best_option(board)
+     win_move(board) || block_move(board) || strategic_move(board)
```
Comment (randallreedjr) [inline lib/players/computer.rb]: Excellent, very readable.

File: lib/tic_tac_toe_cli.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                when "x"
+                  Game.new.play
+                when "o"
+                  Game.new(Player::Human.new("O"), Player::Human.new("X"), Board.new).play
+              end
+            
+              puts "Do you want to play again (y/n)?."
+              play_again = gets.strip
+                
+          elsif kind =="1"
```
Comment (randallreedjr) [inline lib/tic_tac_toe_cli.rb]: Maybe create a separate method for each `kind` to make this a little more manageable.

**Comment by randallreedjr** [comment]
@tejbans Did you read all of my past comments? lol
This was a really great submission, I had almost nothing to say. Keep up the awesome work!

**Comment by randallreedjr** [comment]
Oh, I reviewed you before. Well, great job applying what you learned!

**Comment by tejbans** [comment]
Hi Randall,
 Yes you reviewed me before. I refactored as per your comments .
Thank you! Appreciate your comments.

**Pull request closed without merging**